tests/rados/test_bluestore_features.py

```diff
@@ -96,6 +98,39 @@ def run(ceph_cluster, **kw):
 
     rados_object.change_heap_profiler_state(osd_list, "stop")
 
+    execution_time = config["execution-time"]
+    log.info(f"Checking osd memery growth for {execution_time} minutes")
+    # Setting the scheduled scrub parameters to enable scrubbing
+    if scrub_object.get_osd_configuration("osd_scrub_begin_hour") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_begin_hour", 0)
+        log.info("The osd_scrub_begin_hour is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_end_hour") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_end_hour", 0)
+        log.info("The osd_scrub_end_hour is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_begin_week_day") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_begin_week_day", 0)
+        log.info("The osd_scrub_begin_week_day is set to 0")
+    if scrub_object.get_osd_configuration("osd_scrub_end_week_day") != 0:
+        scrub_object.set_osd_configuration("osd_scrub_end_week_day", 0)
+        log.info("The osd_scrub_end_week_day is set to 0")
+
+    time_execution = datetime.datetime.now() + datetime.timedelta(
+        minutes=int(execution_time)
+    )
+    while datetime.datetime.now() < time_execution:
+        # Get the dump_mempools
+        for osd in osd_list:
+            cmd = f"ceph tell osd.{osd} dump_mempools"
+            out = rados_object.run_ceph_command(cmd)
+            bytes = int(out["mempool"]["total"]["bytes"])
+
+            # convert bytes to gb
+            gb = bytes / (1024 * 1024 * 1024)
+            # Checking the OSD memory is increased or not while scrubbing
+            if gb > 2:
```

Comment:
Check if the increase in memory usage is more than 2 GB. We are checking the increase in memory consumption here.

Reply:
Modified the code.